src/Stratis.Bitcoin.IntegrationTests/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -509,7 +510,21 @@ public bool AddToStratisMempool(Transaction trx)
 
         public List<uint256> GenerateStratisWithMiner(int blockCount)
         {
-            return this.FullNode.Services.ServiceProvider.GetService<IPowMining>().GenerateBlocks(new ReserveScript { ReserveFullNodeScript = this.MinerSecret.ScriptPubKey }, (ulong)blockCount, uint.MaxValue);
+            List<uint256> blocks = new List<uint256>();
+
+            ConsensusLoop consensusLoop = this.FullNode.NodeService<ConsensusLoop>();
+            ConcurrentChain chain = this.FullNode.NodeService<ConcurrentChain>();
+
+            for (int i = 0; i < blockCount; i++)
+            {
+                // Wait here with a small interval. Otherwise PoSMiner will wait for this with 1 minute interval.
```

Comment:
The miner will wait for 1 min?

Reply:
Oh, it's PoW, not PoS. 

PowMining:124

Reply:
fixed the comment (PoS => PoW).